### PR TITLE
Add facility claim data to history response

### PR DESCRIPTION
**Pull request by kellyi**
## Overview

Add claim data to history response including:

- claim was approved
- claim was revoked
- claim had some public details changed (excluding non-public and
potentially non-public fields)

Connects #818 

## Notes

Per our discussion I opted to use a whitelist of public claim update field changes to ensure we don't accidentally serialize something non-public. If demand is there for including potentially non-public fields, we can reassess the approach.

## Testing Instructions

- run tests on this branch and verify that they pass
- serve this branch, turn on all the feature flag switches, then:
   - claim a facility
   - approve the claim
   - make some changes
   - revoke the claim

Then use the Swagger docs to request the facility history and verify that you see each action in the response.

## Checklist

- [ ] `fixup!` commits have been squashed
- [ ] CI passes after rebase
- [x] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines

### CODE REVIEW

File: src/django/api/views.py [hunk 1]
```diff
@@ -789,7 +789,7 @@ def count(self, request):
     @transaction.atomic
     def claim(self, request, pk=None):
         if not switch_is_active('claim_a_facility'):
-            return NotFound()
+            raise NotFound()
```
Comment (kellyi) [inline src/django/api/views.py]: I noticed this mistake while I was writing tests against the claim endpoint.

File: src/django/api/constants.py [hunk 2]
```diff
@@ -49,6 +49,9 @@ class FacilityHistoryActions:
     OTHER = 'OTHER'
     ASSOCIATE = 'ASSOCIATE'
     DISSOCIATE = 'DISSOCIATE'
+    CLAIMED = 'CLAIMED'
```
Comment (jwalgran) [inline src/django/api/constants.py]: Our pattern as been to use a present tense action label. Consider `CLAIM`.

**Comment by kellyi** [comment]
Closing in favor of #852

File: src/django/api/facility_history.py [hunk 3]
```diff
@@ -162,6 +162,60 @@ def processing_results_has_split_action_for_oar_id(list_item, facility_id):
...
+            'facility_parent_company',
+        ]
+
+        return {
+            'updated_at': str(claim.updated_at),
+            'action': FacilityHistoryActions.CLAIM_UPDATE,
+            'detail': 'Facility {} claim public data was updated'.format(
+                claim.facility.id,
+            ),
+            'changes': {
```
Comment (jwalgran) [inline src/django/api/facility_history.py]: If a non-public field is updated, the history record looks like this:

Screen Shot 2019-10-03 at 10 37 29 AM

This looks a bit broken. Consider suppressing the entire history entry if the changes dict is empty.

**Pull request closed without merging**